CMSgov/easi-app
# [EASI-2019] - Intake select email recipients

# EASI-2019

## Changes and Description

- Ability to select multiple email recipients when taking an action
- Can add and select additional recipients
- Toggle ability to show additional optional recipients
- If contact has invalid email, alert is shown below and recipient is disabled
  - Validation will be handled in [EASI-2113](https://jiraent.cms.gov/browse/EASI-2113)
- Cypress tests



## How to test this change
- Test different system intake action types (click "show more recipients" to test additional recipients form)
  - [Request draft business case](http://localhost:3000/governance-review-team/0e305af7-7231-4797-9631-65166087d4a5/actions/need-biz-case)
  - [Close project](http://localhost:3000/governance-review-team/20cbcfbf-6459-4c96-943b-e76b83122dbf/actions/no-governance)
  - [Issue LCID](http://localhost:3000/governance-review-team/af7a3924-3ff7-48ec-8a54-b8b4bc95610b/actions/issue-lcid)
  - [Extend LCID](http://localhost:3000/governance-review-team/d2bfa1ab-b72e-4db4-9b04-6a6100c9d585/actions/extend-lcid)
- All actions should have requester and IT Governance Mailbox as default recipients
- Extend LCID, issue LCID, and close project should also have IT Investment Mailbox as default recipient
- Ensure "recipients selected" count is correct



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
